pyproject.toml

```diff
@@ -33,17 +33,17 @@ dependencies = [
     "joblib>=1.0.0",
     "jupyter",
     "matplotlib",
-    "numpy",
+    "numpy<2.0.0",
     "pillow",
     "pyknos>=0.16.0",
     "pyro-ppl>=1.3.1",
     "scikit-learn",
-    "scipy",
+    "scipy<1.13",
     "tensorboard",
-    "torch>=1.8.0",
+    "torch>=1.13.0",
     "tqdm",
     "pymc>=5.0.0",
-    "zuko>=1.2.0",
+    "zuko>=1.1.0",
```

Comment:
are you sure about this one? I once had to upgrade zuko (but I think I had been on 1.0.0).

Reply:
I think we need a recent one for the `UnconditionalTransform`. 

on the GH page, the most recent release is 1.1.0. But I think it's a typo, the actual version is 1.2

Reply:
yes, we need 1.2.0. Will undo this change.